tests/unit/s2n_server_new_session_ticket_test.c

```diff
@@ -910,7 +988,6 @@ int main(int argc, char **argv)
 
             /* Test with no variable fields */
             {
-                conn->tickets_to_send++;
```

Comment:
What's this removal for?

Reply:
Turning on tickets now automatically configures us to send one ticket. Incrementing this meant that we sent two.